BlueprintUI/Sources/Layout/EqualStack.swift

```diff
@@ -65,7 +65,23 @@ extension EqualStack {
         var spacing: CGFloat
 
         func measure(in constraint: SizeConstraint, items: [(traits: Void, content: Measurable)]) -> CGSize {
-            let itemSizes = items.map { $1.measure(in: constraint) }
+            guard items.count > 0 else { return .zero }
+
+            let totalSpacing = (spacing * CGFloat(items.count - 1))
+            let itemConstraint: SizeConstraint
+            switch direction {
+            case .horizontal:
+                itemConstraint = SizeConstraint(
+                    width: (constraint.width - totalSpacing) / CGFloat(items.count),
```

Comment:
These asymmetrical operators were slightly surprising at first glance but it does seem a lot better than switching inline.